eev.d.ts

```diff
@@ -0,0 +1,13 @@
+declare module 'eev' {
+    type CallbackFunction = (data: any) => void
+
+    export class Eev {
+        constructor();
+        on(names: string[], fn: CallbackFunction): void;
```

Comment:
This should be:

`on(name: string, fn: CallbackFunction): void;`

Honestly, I don't love the way I implemented these functions. If I were to re-write Eev, I'd probably change it to be: `on(name: string | string[], fn: CallbackFunction): Eev` but alas, that's beyond the scope of this pull-request.

Reply:
Ah, it seems I've misread something, will fix it somewhere today. Thanks for taking the time to look at this PR.